woocommerce/woocommerce-android
# Issue/4847 Handling of check mobile device message

Closes: #4847 


### Description
The PR adds handling of `CHECK_MOBILE_DEVICE` event. When this event thrown by the SDK we show `Check your mobile device` message to a user

### Testing instructions
In order to test this, the code has to be modified. Modify `BluetoothReaderListenerImpl:76` with:
```
.CardReaderDisplayMessage(additionalInfoMapper.map(ReaderDisplayMessage.CHECK_MOBILE_DEVICE))
```

Then
* Try to collect a payment with a real reader
* Insert a card inside the reader 
* Notice "Check your mobile device" message is shown

### Images/gif



https://user-images.githubusercontent.com/4923871/134880950-3ac171a7-be7c-4318-9888-091ac4c6d5f9.mp4
